### PR TITLE
add wishlist page

### PR DESCRIPTION
**Pull request by sk-ip**
Wishlist page

### CODE REVIEW

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <div class="badge badge-top-right badge-success">0</div>
+          </div>
+            </a>
+        </div>
+    </div>
+    <div class="wishlist__page">
+
+        <div class="wishlist__list" >
+            <div class="card relative wishlist__card " >
+            <img class="card-image" src="./assets/images/antiglare.jpg" alt="" />
```
Comment (LordUttam) [inline wishlist.html:41]: Write something meaningful in alt attribute.

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            <button class="btn btn-outline-danger wishlist__btn " >Remove</button>
+            <button class="btn btn-primary wishlist__btn" >Add to Cart</button>
+            </div>
+
+            <div class="card relative wishlist__card " >
+            <img class="card-image" src="./assets/images/computer-glasses.jpg" alt="" />
+            <p class="card-subtitle bold ">Name</p>
+            <p class="card-subtitle semibold ">Rs. 1200</p>
+            <button class="btn btn-outline-danger wishlist__btn " >Remove</button>
+            <button class="btn btn-primary wishlist__btn" >Add to Cart</button>
```
Comment (LordUttam) [inline wishlist.html:53]: try to be consistent in naming convention. You seem to have used BEM for wishlist elements but for buttons, modifiers should also have double hyphen i.e. btn--primary.

Comment (LordUttam) [reply wishlist.html:53]: Is it like you used BEM for this project but for CL you are using conventional naming scheme with single hyphens?

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
```
Comment (LordUttam) [inline wishlist.html:8]: Add author meta tag.

File: wishlist.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+.wishlist__list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    padding: 10px;
+}
+
+.wishlist__card {
+    border: 1px solid black;
+    width: 230px;
```
Comment (LordUttam) [inline wishlist.css:16]: Avoid using pixels as pixels are absolute unit. You will face many challanges when you try to make it responsive to different viewports.

**Review by LordUttam — CHANGES_REQUESTED** [review]
Rest looks good

File: wishlist.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
+.wishlist__page {
+    width: 80%;
+    margin: 30px auto;
```
Comment (zabihshaik) [inline wishlist.css:3]: Better to use relative units rems or ems instead of px

File: wishlist.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        <a href="./index.html"><p class="brand heading3">LensMart</p></a>
+        <div class="header__search" >
+            <input type="text" name="" id="">
+            <span class="material-icons search__icon ">search</span>
+        </div>
+        <div class="header__icons" >
+            <a href="./login.html"><button class="btn btn-primary" >Login</button></a>
+            <a href="./wishlist.html">
+                <div class="relative">
+                <span class="material-icons">favorite_border</span>
```
Comment (zabihshaik) [inline wishlist.html:25]: Better to follow consistency in naming

File: wishlist.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+            <div class="card relative wishlist__card " >
+            <img class="card-image" src="./assets/images/computer-glasses.jpg" alt="" />
+            <p class="card-subtitle bold ">Name</p>
+            <p class="card-subtitle semibold ">Rs. 1200</p>
+            <button class="btn btn-outline-danger wishlist__btn " >Remove</button>
+            <button class="btn btn-primary wishlist__btn" >Add to Cart</button>
+            </div>
+
+            <div class="card relative wishlist__card " >
+            <img class="card-image" src="./assets/images/goggles.jpg" alt="" />
```
Comment (zabihshaik) [inline wishlist.html:57]: Try to add something in alt

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. But, some suggestions:
1. Try to add screenshots while opening a PR in description.
2. Better to use meta tag with author and description.
3. Better to follow consistency in naming. Like some places, you have used BEM, and others' different.
4. Try to use relative units like rems and ems instead of absolute units like px.

File: wishlist.html (hunk 2 above)
Comment (sk-ip) [reply wishlist.html:53]: yes that is the case.

File: wishlist.html (hunk 1 above)
Comment (sk-ip) [reply wishlist.html:41]: thanks will make the change

File: wishlist.css (hunk 4 above)
Comment (sk-ip) [reply wishlist.css:16]: yes will make that change

File: wishlist.css (hunk 5 above)
Comment (sk-ip) [reply wishlist.css:3]: yes will do that

**Pull request merged**